source/adapters/hip/common.cpp

```diff
@@ -99,19 +99,18 @@ void checkErrorUR(hipError_t Result, const char *Function, int Line,
     return;
   }
 
-  if (std::getenv("SYCL_PI_SUPPRESS_ERROR_MESSAGE") == nullptr ||
-      std::getenv("UR_SUPPRESS_ERROR_MESSAGE") == nullptr) {
-    const char *ErrorString = nullptr;
-    const char *ErrorName = nullptr;
-    ErrorName = hipGetErrorName(Result);
-    ErrorString = hipGetErrorString(Result);
-    std::cerr << "\nUR HIP ERROR:"
-              << "\n\tValue:           " << Result
-              << "\n\tName:            " << ErrorName
-              << "\n\tDescription:     " << ErrorString
-              << "\n\tFunction:        " << Function
-              << "\n\tSource Location: " << File << ":" << Line << "\n\n";
-  }
+  const char *ErrorString = nullptr;
```

Comment:
Can we initialize this and `ErrorName` with the return vals of `hipGetErrorName` and `hipGetErrorString`?

Reply:
Done